packages/@aws-cdk/assets/lib/fs/copy-options.ts

```diff
@@ -18,3 +18,16 @@ export interface CopyOptions {
    */
   readonly exclude?: string[];
 }
+
+/**
+ * Options related to calculating source hash.
+ */
+export interface FingerprintOptions extends CopyOptions {
```

Comment:
Why did you move this here? Seems like a more natural place for `FingerprintOptions` is the `fingerprint` file where those options are actually being used.

Reply:
Renamed the file to `options.ts`